packages/react-server-examples/bike-share/README.md

```diff
@@ -0,0 +1,65 @@
+# react-server-examples/hello-world
+
+An example project for `react-server` which only server rendering, interactivity on the client side, frameback, ReactServerAgent, url parameters and logging.  Uses [api.citybik.es](http://api.citybik.es/v2/) to get data about bike shares and their availability around the world.
```

Comment:
Looks like there may be a word missing here? "which only server rendering".

Also, please break these long lines up.  Long lines are problematic for [a number of reasons](https://github.com/redfin/react-server/pull/108).